### ISSUE TITLE
"Speed display" redundant options & broken functionality

### ISSUE THREAD

**Issue by Fanatic-64** [project64/project64#1524]
There are a couple of issues with the "Speed display" setting (under Options/Advanced):
1. The "None" option implemented recently on the drop-down menu is redundant since the same effect can be achieved (in theory) by unchecking the checkbox next to the "Speed display" setting.
2. There is a bug however when hiding the speed display by using the checkbox. If the options "None", "Percentage of full speed" or "Vertical interrupts per second" are selected on the drop-down menu, and the checkbox is unchecked, then the speed display will not appear, as intended. However, if the option "Display lists per second" is selected, then the speed display will appear even if the checkbox is unchecked, and if the option "VI/s & DL/s" is selected, then the speed display will also appear even with the checkbox unchecked, but the VI/s will remain at 0.0 while the DL/s will display normally.

**Comment by project64** [comment]
you are right, I removed the none option https://github.com/project64/project64/commit/9ab7a83c2686ee8e160fda22452de38aa986e7a1

**Comment by theboy181** [comment]
Since this change the VI/s % DL/s setting no longer displays correctly the VI/s when this option is selected. Something is not right here. @project64 how should it be? 

Reproduced with fresh install, and only changing the pull down while not having Speed display checked.

![image](https://user-images.githubusercontent.com/9299743/50490791-dba7f100-09c3-11e9-849c-86851beb8fb2.png)

**Comment by project64** [comment]
I am not sure what your talking about. what are you expecting to happen that is not to make you think it is not right?

**Comment by theboy181** [comment]
Did you clone the settings? 

When you have just vi and dl together only the vi will say 0 and the dl will display the dl. You have to select speed display for it to work correctly. Jump on discord and I can share my screen if you want a visual of it

**Comment by Fanatic-64** [comment]
^I believe this is the same issue I described in my original post:

>There is a bug however when hiding the speed display by using the checkbox. If the options "None", "Percentage of full speed" or "Vertical interrupts per second" are selected on the drop-down menu, and the checkbox is unchecked, then the speed display will not appear, as intended. However, if the option "Display lists per second" is selected, then the speed display will appear even if the checkbox is unchecked, and if the option "VI/s & DL/s" is selected, then the speed display will also appear even with the checkbox unchecked, but the VI/s will remain at 0.0 while the DL/s will display normally.
